client/src/components/content/RobotContainer/CreateRobotContainer.jsx

```diff
@@ -14,30 +13,34 @@ const { Title } = Typography;
  * @category Client
  */
 const CreateRobotContainer = (props) => {
-
-    const addRobot = () => alert('Unfortunately this is currently not supported.'); //props.createNewRobot();
-
-    return (
-        <Col xs={24} sm={12} md={8} xl={6} xxl={4}>
-            <Col className={[styles.box, styles.createRobotBox]} >
-                <Row align="middle" style={{ height: '70%' }}>
-                    <Col type="flex" span={24}>
-                        <PlusCircleOutlined onClick={addRobot} style={{ fontSize: '6rem', color: corporateDesign.colorBackground }} />
-                    </Col>
-                </Row>
-
-                <Row justify="space-around" align="middle" style={{ height: '40%' }}>
-                    <Title className={styles.title} level={3} >
-                        Create new Robot
-                    </Title>
-                </Row>
-            </Col>
-        </Col >
-    )
+  const addRobot = () => props.createNewRobot();
```

Comment:
I know it is legacy code but it is better to use explicit properties for the class. So instead of props to use {createNewRobot} and than to use the function createNewRobot() without the props. before :)